README.md

```diff
@@ -279,6 +280,18 @@ BAR=FOO
 FOO=BAR
 ```
 
+### Quote Character
```

Comment:
```suggestion
### Quote Character (Will be available in v2.0.0)
```
Also, please add the additional information that this check will be available in v2.0.0.